conftest.py

```diff
@@ -140,7 +140,13 @@ def pytest_addoption(parser):
         type=str,
         help="Select the containerizer used to launch the Jenkins master.",
     )
-
+    parser.addoption(
```

Comment:
Is this really required? I don't mind but the service configures two Jenkins agent labels anyways.